.github/workflows/ci.yml

```diff
@@ -311,10 +311,9 @@ jobs:
       - name: Report to codecov.io
         uses: codecov/codecov-action@v4
         with:
-          token: ${{ secrets.CODECOV_UPLOAD_TOKEN }}
+          token: ${{ secrets.CODECOV_TOKEN }}
           files: ./lcov.info
-          fail_ci_if_error: true
-          verbose: true
+          fail_ci_if_error: false
```

Comment:
nit: this defaults to `false`, so maybe better to just omit it?

Reply:
@ctz Any feelings on this? If you prefer it being explicit I don't care enough to block merging :-)

Reply:
Sorry, missed that comment. I prefer to have it explicit, as I find the default pretty surprising.

Reply:
SGTM